tools/bake.py

```diff
@@ -718,6 +719,9 @@ def perform_bake(self, context):
         cleanup_shapekeys = context.scene.bake_cleanup_shapekeys # Reverted and _old shapekeys
         ignore_hidden = context.scene.bake_ignore_hidden
 
+        #adjusting don't pack to disable if there are things that will cause undesirable results if enabled, then invert and store to pack_uvs var
+        pack_uvs = not (dont_pack_uvs and (not emit_indirect) and (not prioritize_face) and (not pass_ao) and (not any(plat.use_decimation for plat in context.scene.bake_platforms)))
```

Comment:
This should just be "not dont_pack_uvs", if someone has explicitly selected "don't pack" then we should honor that, even if it'll have bad results.